scripts/data/src/org/akvo/gae/remoteapi/FindFormsInFolder.java

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright (C) 2021 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.gae.remoteapi;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class FindFormsInFolder implements Process {
+    @Override
+    public void execute(DatastoreService ds, String[] args) throws Exception {
+        String folderName = args[0];
+        long folderId = findFolderId(ds, folderName);
+        List<Entity> forms = findFormsInFolder(ds, folderId);
+        Map<Long, Entity> surveys = findAssociatedSurveys(ds, forms);
+        printFormIds(forms, surveys);
+        Entity one = new Entity("Survey");
```

Comment:
Is this used or initialized for some reason? I don't see it used later or is it the first part of something?

Reply:
oops. that should not be there.